src/app/domain/case-viewer/components/case-viewer/case-viewer.component.ts

```diff
@@ -6,8 +6,9 @@ import {Component, Input, OnChanges} from '@angular/core';
     styleUrls: ['./case-viewer.component.scss']
 })
 export class CaseViewerComponent implements OnChanges {
-    @Input() case: any;
 
+    @Input() case: any;
```

Comment:
What are these for?

Reply:
`+    @Input() case: any;`
was added by you 
`    @Input() caseId: any;` been added I guess because you need the id to link to the timeline
I am sure there other way to do that which will be done in my work with router in the near future so for now could you just leave this please else require me to do double the work in two branches